src/controller/python/chip-device-ctrl.py

```diff
@@ -882,6 +882,7 @@ def start_rpc_server():
         server.register_function(get_pase_data)
         server.register_function(get_fabric_id)
         server.register_multicall_functions()
+	server.register_introspection_functions()
```

Comment:
Indentation seems off here on GitHub. Is it a mix of spaces and tabs?

Reply:
somehow changes from local machine to Raspi were not saved. Resolved alignment issue